### PR TITLE
BlahBlah Test dont merge (sui-nyan)

### PR DESCRIPTION
**Pull request by kida31**

### CODE REVIEW

File: Fish.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+using Godot;
+using System;
+
+public partial class Fish : Area2D
+{
+	public float BaseSpeed { get; set; } = 200;
```
Comment (kida31) [inline Fish.cs:6]: Set accessors are not consistent.
Please revise and group/sort code order by access

File: Hookbar.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
+using Godot;
+using System;
+
+public partial class Hookbar : Area2D
+{
+	private Fish _hookedFish;
+	public float StepDistance { get; set; }= 100;
```
Comment (kida31) [inline Hookbar.cs:7]: Revise accessors to minimum permission.

File: Hookbar.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+		{
+			ds = direction * StepDistance;
+		}
+
+		GlobalPosition += ds;
+		GD.Print("Hookbar moved");
+	}
+
+	private void OnAreaEntered(Area2D area)
+	{
```
Comment (kida31) [inline Hookbar.cs:75]: Replace GD.Print with logger

File: Fish.cs (hunk 1 above)
Comment (Sui-nyan) [reply Fish.cs:6]: fixed

File: Hookbar.cs (hunk 2 above)
Comment (Sui-nyan) [reply Hookbar.cs:7]: fixing bugs and creating a new one

File: Hookbar.cs (hunk 3 above)
Comment (Sui-nyan) [reply Hookbar.cs:75]: remove print

File: FishingGame.cs [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+	[Export] private LineEdit _hookDistanceMod;
+	[Export] private LineEdit _hookDistance;
+	[Export] private LineEdit _progressDecayPerSecondText;
+	[Export] private LineEdit _progressPerPullText;
+
+
+	// Called when the node enters the scene tree for the first time.
+	public override void _Ready()
+	{
+		_hookbar.PulledFish += OnFishPulled;
```
[the remark this answers is not in the document]
Comment (Sui-nyan) [reply FishingGame.cs:26]: 🪨

[the remark this answers is not in the document]
Comment (kida31) [reply FishingGame.cs:26]: No, this has not been implemented yet

File: Hookbar.cs (hunk 3 above)
Comment (kida31) [reply Hookbar.cs:75]: Dont remove it, keep it, but use logger instead

File: Hookbar.cs [hunk 5]
```diff
@@ -74,12 +73,13 @@ private void Move(Vector2 direction)
...
 	{
-		if (area is Fish fish)
+		if (area is not Fish fish)
 		{
-			GD.Print("hooked fish");
-			_hookedFish = fish;
-			fish.IsHooked = true;
+			return;
 		}
+
```
Comment (kida31) [inline Hookbar.cs:80]: i dont like this new line. Please remove

File: Hookbar.cs (hunk 3 above)
Comment (kida31) [reply Hookbar.cs:75]: suinyan: fixed

File: Hookbar.cs (hunk 2 above)
Comment (kida31) [reply Hookbar.cs]: yeah, but bugs are bad

**Pull request closed without merging**